unit-tests-jest/numbers.test.ts

```diff
@@ -0,0 +1,51 @@
+import { evenNumbers, toDollars, divideBy, multiplyBy } from './numbers';
+
+describe('evenNumbers', () => {
+  it('returns the even numbers', () => {
+    const numbers = [1, 4, 5, 10, 0, 7.4, -8];
+    const result = evenNumbers(numbers);
+    expect(result).toEqual([4, 10, 0, -8]);
+  });
+});
+describe('evenNumbers', () => {
```

Comment:
Usually you will put all the 'it' or 'test' functions inside the same `describe`. Another way to think about that is that there should be only one `describe` for each function you are testing.